api/server/server.go

```diff
@@ -1482,27 +1550,31 @@ func createRouter(s *Server) *mux.Router {
 		ProfilerSetup(r, "/debug/")
 	}
 	m := map[string]map[string]HttpApiFunc{
+		"HEAD": {
+			"/containers/{name:.*}/archive-path": s.getContainersArchivePath,
```

Comment:
There is no HEAD handler, instead the server will serve the GET response minus the body.